Sources/Resolver/Resolver.swift

```diff
@@ -592,6 +592,9 @@ public struct InjectedObject<Service>: DynamicProperty where Service: Observable
     public init(name: String? = nil, container: Resolver? = nil) {
         self.service = container?.resolve(Service.self, name: name) ?? Resolver.resolve(Service.self, name: name)
     }
+    public init(_ args: Any) {
```

Comment:
If this were going to be added it would need to be an optional parameter on the existing initializers, and not an isolated init method that doesn't allow name or container to be specified.